src/main/flight/imu.c

```diff
@@ -723,3 +723,18 @@ bool isUpright(void)
     return true;
 #endif
 }
+
+
+#if defined(USE_ALTHOLD_MODE)
```

Comment:
This can be unconditional

Reply:
It's currently used only in alt-hold. And we have a flash-space related issues.

Reply:
LTO/linker will get rid of unused function

Reply:
Fixed.